docs/user/tutorial/70-page-management.rst

```diff
@@ -4,7 +4,10 @@ Managing pages
 
 Our menu of pages is growing.
 
-.. todo:: screenshots of menu
+|page_tree|
```

Comment:
By the way you don't need to do this - it's only necessary to create a ``|substitution|`` when you want an inline image.

For an image as a block, just do:

```
.. image:: /path/image.url
    etc etc
```